recOrder/cli/apply_inverse_transfer_function.py

```diff
@@ -27,23 +28,55 @@ def _check_background_consistency(background_shape, data_shape):
 
 
 def apply_inverse_transfer_function_cli(
-    input_data_path, transfer_function_path, config_path, output_path
+    input_data_path: str,
+    transfer_function_path: str,
+    configuration_settings: Union[str, ReconstructionSettings],
+    output_path: str,
 ):
+    """
+    1) Reads an input dataset and transfer function from zarr stores,
+    2) Applies the inverse transfer function using settings from a configuration yaml file, then
+    3) Saves the result to an output zarr store.
+
+    This method also supports configuration settings passed directly from a
+    ReconstructionSettings object for live reconstruction purposes.
+
+    Parameters
+    ----------
+    input_data_path : str
+        Path to position-level input zarr store
+    transfer_function_path : str
+        Path to transfer function zarr store
+    configuration_settings : Union[str, ReconstructionSettings]
+        Path to configuration settings yaml file, or
+        directly from a ReconstructionSettings object
+    output_path : str
+        Path to output zarr store
+    """
     echo_headline("Starting reconstruction...")
 
     # Load datasets
     transfer_function_dataset = open_ome_zarr(transfer_function_path)
     input_dataset = open_ome_zarr(input_data_path)
 
-    # Load config file
-    settings = utils.yaml_to_model(config_path, ReconstructionSettings)
+    # Load configuration settings file
```

Comment:
One point we talked about offline is to put this block of code in its own function, since this function is already too long.